case_studies/sdss_galaxies_vae/reconstruction.py

```diff
@@ -66,13 +66,13 @@ def reconstruct(cfg):
         print(
             f"{(tile_map_recon['galaxy_blends'] > 1).sum()} galaxies are part of blends in image."
         )
-        map_recon = get_full_params_from_tiles(tile_map_recon, encoder.tile_slen)
+        map_recon = tile_map_recon.to_full_params()
         map_recon["fluxes"] = (
             map_recon["galaxy_bools"] * map_recon["galaxy_fluxes"]
             + map_recon["star_bools"] * map_recon["fluxes"]
         )
         map_recon["mags"] = convert_flux_to_mag(map_recon["fluxes"])
-        map_recon["plocs"] = map_recon["plocs"] - 0.5
+        # map_recon["plocs"] = map_recon["plocs"] - 0.5
```

Comment:
Why don't you need the `-0.5` anymore ?

Reply:
I think I accidently included this change from #484 - I was experimenting trying to align the semi-synthetic coadd catalog. I'll remove it.